subsystem/machine_pool_resource_test.go

```diff
@@ -662,11 +662,23 @@ var _ = Describe("Machine pool creation", func() {
 					}
 				  },
 				  "replicas": 12,
-                  "labels": {}
+                  "labels": null
 				}`),
 			),
 		)
 
+		// Invalid deletion - labels map can't be empty
+		terraform.Source(`
+		  resource "rhcs_machine_pool" "my_pool" {
+		    cluster      = "123"
+		    name         = "my-pool"
+		    machine_type = "r5.xlarge"
+		    replicas     = 12
+            labels       = {}
+		  }
+		`)
+		Expect(terraform.Apply()).ToNot(BeZero())
+		// Valid deletion
```

Comment:
Please also add an update flow:
1. Create MP with labels 
2. remove them by setting it to empty map -> should fail 
3. Remove them by setting it to null -> should succeed

Reply:
That is what I am doing in the test. 🤔  

in tf `=null` is like not passing the attribute. I can add that case but seems pointless. wdyt?